tests/unit/modules/test_calendar_extraction.py

```diff
@@ -5,29 +5,30 @@
 import xarray as xr
 
 from pywatts.core.exceptions.wrong_parameter_exception import WrongParameterException
-from pywatts.modules.calendar_extraction import CalendarExtraction
+from pywatts.modules.calendar_extraction import CalendarExtraction, CalendarFeature
 
 
 class TestCalendarExtraction(unittest.TestCase):
     def setUp(self):
         start_time = "2000-01-01"
         times = 365
         self.da = xr.DataArray(data=pd.date_range(start_time, freq="D", periods=times))
-        self.calendar_features = ["year", "month", "day", "weekday", "hour", "weekend", "workday", "holiday"]
+        self.calendar_features = [CalendarFeature.year, CalendarFeature.month, CalendarFeature.day,
+                                  CalendarFeature.weekday, CalendarFeature.hour, CalendarFeature.workday,
+                                  CalendarFeature.holiday, CalendarFeature.hour_sine, CalendarFeature.hour_cos,
+                                  CalendarFeature.weekday_sine, CalendarFeature.weekday_cos, CalendarFeature.month_sine,
+                                  CalendarFeature.month_cos, CalendarFeature.day_sine, CalendarFeature.day_sine]
```

Comment:
2x day_sine

do we want to test all features, including Monday, Tuesday, etc., or only a representative sample?